konfera/views.py

```diff
@@ -1,3 +1,16 @@
-from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404
+from konfera.models import Event, Sponsor
```

Comment:
Please check the result of automated test, if this is not the same problem as mentioned above? Or you can test locally if you run server works?

If it will be a problem you can allways import like this:
`from konfera.models.event import Event` and it will work...